spec/helpers/course/controller_helper_spec.rb

```diff
@@ -19,7 +19,7 @@
     end
 
     describe '#link_to_course_user' do
-      let(:user) { build(:course_user) }
+      let(:user) { create(:course_user) }
```

Comment:
Yup - since `link_to_course_user` redirects to `course_user_path` which resolves to `/course/:course_id/user/:id`, the record must be created and saved for a course_user id, course_user_id is nil.